### PR TITLE
mod : updated itemvaluemodel for crossbowrework

### PR DESCRIPTION
**Pull request by namidaka**

### CODE REVIEW

File: src/Module.Server/Common/Models/CrpgItemValueModel.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public override float CalculateTier(ItemObject item)
         return item.ItemComponent switch
         {
             ArmorComponent armorComponent => CalculateArmorTier(armorComponent),
-            HorseComponent horseComponent => CalculateHorseTier(item, horseComponent), // horseComponent.Item throws so item needs to be passed here
+            HorseComponent horseComponent => CalculateHorseTier(item, horseComponent),
```
Comment (verdie-g) [inline src/Module.Server/Common/Models/CrpgItemValueModel.cs:38]: To discard

File: src/Module.Server/Common/Models/CrpgItemValueModel.cs [hunk 2]
```diff
@@ -424,16 +424,25 @@ private float ItemToHeirloomLevel(ItemObject item)
     {
         if (item == null)
         {
+            InformationManager.DisplayMessage(new InformationMessage("one of the item was null, please report to namidaka"));
             return 0;
         }
 
-        if (!item.StringId.StartsWith("crpg_")
```
Comment (verdie-g) [inline src/Module.Server/Common/Models/CrpgItemValueModel.cs:430]: Looks like you are reverting the last commit here

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/Module.Server/Common/Models/CrpgItemValueModel.cs (hunk 1 above)
Comment (namidaka) [reply src/Module.Server/Common/Models/CrpgItemValueModel.cs]: done

File: src/Module.Server/Common/Models/CrpgItemValueModel.cs (hunk 2 above)
Comment (namidaka) [reply src/Module.Server/Common/Models/CrpgItemValueModel.cs]: solved

File: src/Module.Server/Common/Models/CrpgItemValueModel.cs [hunk 3]
```diff
@@ -435,5 +435,4 @@ private float ItemToHeirloomLevel(ItemObject item)
 
         return heirloomLevel;
     }
-
 }
```
Comment (namidaka) [inline src/Module.Server/Common/Models/CrpgItemValueModel.cs:438]: this is just format correcting

**Review by verdie-g — APPROVED** [review]

**Pull request merged**